torch_geometric/graphgym/model_builder.py

```diff
@@ -19,12 +23,35 @@
 class GraphGymModule(LightningModule):
     def __init__(self, dim_in, dim_out, cfg):
```

Comment:
maybe we add some typing here? I wouldn't know what `cfg` is

Reply:
thank you for reviewing the PR! I have updated this :)